conbench/templates/app.html

```diff
@@ -102,7 +102,13 @@ <h4 class="modal-title"><span class="glyphicon glyphicon-trash submenu"></span>&
 
         {# application content needs to be provided in the app_content block #}
         {% block app_content %}{% endblock %}
+    <br>
+    <hr>
+    <div style="display:inline-block; float: right; color: #918d8c;">
+        Conbench {{ version_string_footer }} | <a href="/api/docs" target="_blank">api docs</a>
```

Comment:
This adds a link to API docs into the footer -- want to remove it from the navbar (see #839).

Reply:
love this!